kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/messages/MessageFormat.kt

```diff
@@ -0,0 +1,22 @@
+package com.clouway.kcqrs.core.messages
+
+import java.io.InputStream
+import java.lang.reflect.Type
+
+/**
+ * JsonFormat is an abstract JSON message format used for parsing and serializing of input messages.
+ * 
+ * @author Miroslav Genov (miroslav.genov@clouway.com)
+ */
+interface MessageFormat {
```

Comment:
I think this interface should be renamed to  ``` MessageFormater ``` and it will look like this ``` messageFormater.format("some event") ``` . 
Also it parse the messages too, that makes me think ``` MessageTransformer ``` will be a better name.

Reply:
http://www.carlopescio.com/2011/04/your-coding-conventions-are-hurting-you.html
http://objology.blogspot.bg/2011/09/one-of-best-bits-of-programming-advice.html
http://www.yegor256.com/2015/03/09/objects-end-with-er.html